ovirtlago/virt.py

```diff
@@ -372,15 +372,14 @@ def engine_setup(self, config=None):
             raise RuntimeError('Failed to setup the engine')
 
     @require_sdk(version='4')
-    def _search_vms(self, api, query):
-        vms_service = api.system_service().vms_service()
+    def _search_vms(self, vms_service, query):
         return [vm.id for vm in vms_service.list(search=query)]
 
     @require_sdk(version='4')
     def start_all_vms(self, timeout=8 * 60):
         api = self.get_api_v4(check=True)
         vms_service = api.system_service().vms_service()
-        ids = self._search_vms(api, query='status=down')
+        ids = self._search_vms(vms_service, query='status=down')
```

Comment:
Well, the assumption is that the HE VM is already up - otherwise, who is this API talking to?
(It might need special treatment, but in a different patch).

Reply:
good point.